addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -125,8 +125,10 @@ $o-we-zindex: $o-we-overlay-zindex + 1 !default;
         outline: none !important;
     }
     .dropdown-menu {
+        transform: none !important;
```

Comment:
Commit message: "make dropdown menus should scrollable " is not proper english ;)